src/python/example/cech_complex_sklearn_itf.py

```diff
@@ -31,8 +31,9 @@
 X = []
 
 for idx in range(dataset_size):
-    pts = points.sphere(nb_points[idx], 2, radius=target[idx])
-    ns = noise * target[idx] * np.random.rand(nb_points[idx], 2) - (noise / 2.)
+    radius = target[idx][0]
```

Comment:
```suggestion
    radius = target[idx, 0]
```
may be more idiomatic in Python?